### PR TITLE
bybit: amount, price and cost precision helper methods

### PR DESCRIPTION
**Pull request by Dan-krm**
Edited createOrder and editOrder so that amountToPrecision, priceToPrecision and costToPrecision aren't called if the precisions are not available. Hard coded some amount and price precisions for expired option markets.

### CODE REVIEW

File: ts/src/bybit.ts [hunk 1]
```diff
@@ -3565,7 +3578,12 @@ export default class bybit extends Exchange {
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
          */
         await this.loadMarkets ();
-        const market = this.market (symbol);
+        let market = this.market (symbol);
+        if (market['option']) {
```
Comment (carlosmiei) [inline ts/src/bybit.ts:3582]: @Dan-krm no we can't do that, that will introduce a huge unexpected delay upon placing the order.

File: ts/src/bybit.ts [hunk 2]
```diff
@@ -4192,10 +4203,15 @@ export default class bybit extends Exchange {
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
          */
         await this.loadMarkets ();
+        let market = this.market (symbol);
+        if (market['option']) {
+            this.options['loadAllOptions'] = true;
```
Comment (carlosmiei) [inline ts/src/bybit.ts:4208]: same here

File: ts/src/bybit.ts (hunk 1 above)
Comment (Dan-krm) [reply ts/src/bybit.ts]: Okay, I also hard coded the amount precision and price precision based on the base currency in createExpiredOption which prevents the issue of having undefined values.

I adjusted the conditional to only set loadAllOptions to true if the values are also undefined or we can remove that all together

File: ts/src/bybit.ts (hunk 2 above)
Comment (carlosmiei) [reply ts/src/bybit.ts:4214]: @Dan-krm this is the same thing 😅 we cannot do things like that
we should:
- use amount/price to precision is precision is available
- don't format price/amount precision if those values are not available

Comment (Dan-krm) [reply ts/src/bybit.ts]: Okay it should be good now

**Pull request merged**